src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php

```diff
@@ -527,6 +527,11 @@ public function delete($id)
             throw new CollectionNotFoundException($id);
         }
 
+        // SQlite does not support cascade-delete, so manually delete the medias.
+        foreach ($collectionEntity->getMedia() as $media) {
+            $this->em->remove($media);
+        }
```

Comment:
Remove this, we are not going to emulate the RDBMS - although maybe not required as I think the ORM should be able to "soft cascade": `cascade: [ "delete" ]`, perhaps.